openforcefield/utils/serialization.py

```diff
@@ -120,6 +120,10 @@ def to_json(self, indent=None):
         import json
 
         d = self.to_dict()
+        # TODO: More generally check for bytes in dict
+        if "conformers" in d.keys():
+            d = _prep_numpy_data_for_json(d)
```

Comment:
This is only a stopgap solution. It may be better to more generally check for the type of the data, but this field is the only one in the toolkit that currently causes this issue.